modules/programs/antidote.nix

```
@@ -33,21 +34,29 @@ in {
 
   config = mkIf cfg.enable {
     home.packages = [ cfg.package ];
-
-    home.file."${relToDotDir ".zsh_plugins.txt"}".text = zPluginStr cfg.plugins;
-
-    ### move zsh_plugins.txt
-    programs.zsh.initExtraBeforeCompInit = ''
-      ## home-manager/antidote begin :
-      source ${cfg.package}/share/antidote/antidote.zsh
-      ${optionalString cfg.useFriendlyNames
-      "zstyle ':antidote:bundle' use-friendly-names 'yes'"}
-      bundlefile=$HOME/${relToDotDir ".zsh_plugins.txt"}
-      zstyle ':antidote:bundle' file $bundlefile
-      staticfile=$HOME/${relToDotDir ".zsh_plugins.zsh"}
-      zstyle ':antidote:static' file $staticfile
-      antidote load $bundlefile $staticfile
-      ## home-manager/antidote end
-    '';
+    programs.zsh.initExtraBeforeCompInit = (
+
```

No need for the parentheses
```suggestion
    programs.zsh.initExtraBeforeCompInit =
```